src/main/java/org/ambraproject/wombat/service/remote/SolrQueryBuilder.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2017 Public Library of Science
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package org.ambraproject.wombat.service.remote;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+import org.ambraproject.wombat.util.UrlParamBuilder;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.http.NameValuePair;
+
+public class SolrQueryBuilder {
+  public static List<NameValuePair> buildParameters(ArticleSearchQuery asq) {
+    UrlParamBuilder params = UrlParamBuilder.params();
+
+    params.add("wt", "json");
+
+    if (asq.isPartialSearch()) {
+      params.add("qf", "doc_partial_body");
+      params.add("fl", "*");
+      params.add("fq", "doc_type:partial");
+    } else {
+      params.add("fq", "doc_type:full");
+    }
+
+    params.add("fq", "!article_type_facet:\"Issue Image\"");
```

Comment:
We do this for saved searches too.  When is this facet wanted I wonder? I also wonder if solr could be configured to skip this facet by default.

Reply:
Perhaps we should stop indexing issue images? If we always want to exclude them, the simplest thing would be to ensure they aren't indexed :)

Reply:
They are still indexed http://solr-mega-prod.soma.plos.org/solr/journals_prod_20191101/select?fq=article_type_facet:%22Issue%20Image%22&q=*:*